### PR TITLE
fix Multiple While

### PR DESCRIPTION
**Pull request by lukyanoffpashok**

### CODE REVIEW

File: test/patterns/multiple_while/test_multiple_while.py [hunk 1]
```diff
@@ -33,7 +33,37 @@ def test_simple(self):
...
                     os.path.realpath(__file__)
-                ).parent, 'MultipleWhile.java'
+                ).parent, 'Simple.java'
+            )
+        )
+        self.assertEqual(lines, [2])
+
+    def test_one_while(self):
+        lines = MultipleWhile().value(
+            Path(
```
Comment (aravij) [inline test/patterns/multiple_while/test_multiple_while.py:43]: Here and everywhere else we better write
```python
Path(__file__).parent.absolute() / 'OneWhile.java'
```

File: aibolit/patterns/multiple_while/multiple_while.py [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
         pass
 
-    def value(self, filename: str):
+    def get_number_of_sequential_while_statement_in_function(self, tree: AST, node: int) -> int:
+        list_while_nodes: List[int] = []
+        set_child_while_nodes: Set[int] = set()
+        for child in tree.all_children_with_type(node, ASTNodeType.WHILE_STATEMENT):
+            list_while_nodes.append(child)
+            set_internal_while = set(tree.list_all_children_with_type(child, ASTNodeType.WHILE_STATEMENT))
+            set_child_while_nodes = set.union(set_child_while_nodes, set_internal_while)
```
Comment (aravij) [inline aibolit/patterns/multiple_while/multiple_while.py:17]:
```suggestion
            set_child_while_nodes |= set_internal_while
```

File: aibolit/patterns/multiple_while/multiple_while.py [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
+from typing import List, Set
 
 
 class MultipleWhile:
 
     def __init__(self):
         pass
 
-    def value(self, filename: str):
+    def get_number_of_sequential_while_statement_in_function(self, tree: AST, node: int) -> int:
```
Comment (aravij) [inline aibolit/patterns/multiple_while/multiple_while.py:11]: Maybe call it `get_top_level_while_qty`?

Comment (lukyanoffpashok) [reply aibolit/patterns/multiple_while/multiple_while.py]: ok

File: aibolit/patterns/multiple_while/multiple_while.py (hunk 2 above)
Comment (lukyanoffpashok) [reply aibolit/patterns/multiple_while/multiple_while.py]: ok

File: test/patterns/multiple_while/test_multiple_while.py (hunk 1 above)
Comment (lukyanoffpashok) [reply test/patterns/multiple_while/test_multiple_while.py]: ok

File: test/patterns/multiple_while/MultipleWhile.java [hunk 4]
```diff
@@ -1,11 +1,11 @@
 class MultipleWhile {
   void bar() {
     while (true) {
-      // some code
+      x = 1;
```
[the remark this answers is not in the document]
Comment (lukyanoffpashok) [reply test/patterns/multiple_while/MultipleWhile.java]: It does not affect

**Comment by aravij** [comment]
@acheshkov please merge

**Comment by acheshkov** [comment]
@rultor merge

**Pull request merged**